bin/build

```diff
@@ -1,13 +1,29 @@
 #!/usr/bin/env bash
 
+DEBUG=("$1" == "debug")
+
 set -e
 
+cd $(dirname $0)/..
+
 . bin/build_utils
 
 FULL_VERSION_TAG="$(full_version_tag)"
 
 echo "---"
 
+if [ $DEBUG ]; then
+
+  echo "Building secrets-provider-debug Docker image"
+
+  docker build \
+    --tag "secrets-provider-debug:latest" \
+    --target "secrets-provider-debug" \
+    .
+
+  exit 0
```

Comment:
can we please not exit? how about putting this and the rest of the logic in an "if-else"?

what i'm afraid is that we will add more logic below that will be missed with this exit. if it's in an "if-else" then it's harder to miss in the future.